pkg/wallet/options.go

```diff
@@ -281,6 +284,13 @@ func AddByCollection(collectionID string) AddContentOptions {
 	}
 }
 
+// WithDataModelValidation enables or disables data validations of adding content.
+func WithDataModelValidation(validateDataModel bool) AddContentOptions {
```

Comment:
In case of boolean option, you don't have to pass any argument. 
Follow [this](https://github.com/hyperledger/aries-framework-go/blob/d5971cedf7bac70db39048faf96c087ba1fbcc40/pkg/doc/verifiable/credential.go#L590-L594) example.

Something like `ValidateContent()`, by default it false and when someone uses this option then it becomes true.